src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strWithStyles = sourceString.split(';');
+  const objectCss = {};
+
+  strWithStyles.forEach((rule) => {
+    const [key, value] = rule.split(':').map((part) => part.trim());
+
+    if (key && value) {
+      objectCss[key] = value;
+    }
+  });
+
+  return objectCss;
```

Comment:
The function `convertToObject` correctly splits the string by semicolons and iterates over the resulting array to create an object. However, it may be beneficial to chain the `split` and `map` operations for better readability. Additionally, consider using more descriptive variable names than `strWithStyles` and `objectCss` to improve code clarity.